files/docker-container-logrotate

```diff
@@ -1,9 +1,11 @@
 /var/lib/docker/containers/*/*.log {
   rotate 0
-  daily
+  hourly
```

Comment:
Can't really predict the growth right now, if something goes rough, dns logs might fill up very fast. Irrespective of that, even on regular basis, logs fill out fast reaching the 50gb limit and making the node unreachable. Once node is out nothing gets logged so we can't predict really at this point. Hourly rotation should give us better idea.